external/corefx-bugfix/src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -0,0 +1,803 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO.Enumeration;
+using System.Threading;
+using System.Threading.Tasks;
+
+#if MONO && (!MOBILE || MOBILE_DESKTOP_HOST || UNITY_AOT)
```

Comment:
Did you make any other change in this file?  Did you annotate it?

Reply:
I had done this as separate commit but forgot and squashed. Let me diff and verify.

Reply:
Pushed as two separate commits to see changes.